### PR TITLE
docs: 132 improve docs

### LINKED ISSUE #132: docs
### Description

A proper doc for colliders, maybe handle here too more of the docs

### Suggested solution

follow the standard

### Alternative

_No response_

### Additional context

_No response_

### Validations

- [X] I agree to follow this project's [Code of Conduct](https://github.com/Tresjs/rapier/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/Tresjs/rapier/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://rapier.tresjs.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/tresjs/rapier/issues) that reports the same bug to avoid creating a duplicate.

### PR DESCRIPTION
**Pull request by JaimeTorrealba**
closes #132

### CODE REVIEW

File: docs/components/rigid-body.md [hunk 1]
```diff
@@ -1,204 +1,230 @@
...
+use[Template ref](https://vuejs.org/guide/essentials/template-refs.html#template-refs).
+:::
+
+## Expose object
+
+```js
+const exposeObject = {
+  instance: rigidBodyInstance,
+  rigidBodyDesc,
+  context: colliderInfos,
```
Comment (Neosoulink) [inline docs/components/rigid-body.md:227]: Now I'm thinking to rename it `colliders` instead of `context`

**Review by Neosoulink — APPROVED** [review]
Wow, didn't expect this doc to be that big and so well done 🚀

File: docs/guide/index.md
Comment (alvarosabu) [inline docs/guide/index.md:1]: Way cleaner 👏🏻

File: docs/components/rigid-body.md [hunk 2]
```diff
@@ -1,204 +1,230 @@
...
+
+:::info
+The information in this page is a summary of the RigidBody instance,
+please check the
+[complete documentation](https://rapier.rs/docs/user_guides/javascript/rigid_bodies)
+:::
+
+## Basic usage
+
+To use a `RigidBody` component, the best case is to import it from
```
Comment (alvarosabu) [inline docs/components/rigid-body.md:18]:
```suggestion
To use a `RigidBody` component, import it from
```

File: docs/components/rigid-body.md [hunk 3]
```diff
@@ -1,204 +1,230 @@
...
+`@tresjs/rapier` and then pass as
```
Comment (alvarosabu) [inline docs/components/rigid-body.md:19]:
```suggestion
`@tresjs/rapier` and then pass as the element you want tot attach via
```

File: docs/components/rigid-body.md [hunk 4]
```diff
@@ -1,204 +1,230 @@
...
+[slot](https://vuejs.org/guide/components/slots.html#scoped-slots) the element
```
Comment (alvarosabu) [inline docs/components/rigid-body.md:20]:
```suggestion
[slot](https://vuejs.org/guide/components/slots.html#scoped-slots).
```

File: docs/components/rigid-body.md [hunk 5]
```diff
@@ -1,204 +1,230 @@
...
+that you want to attach.
```
Comment (alvarosabu) [inline docs/components/rigid-body.md:21]:
```suggestion
```

File: docs/components/rigid-body.md [hunk 6]
```diff
@@ -1,204 +1,230 @@
...
+      <TresTorusGeometry />
+      <TresMeshNormalMaterial />
+    </TresMesh>
+  </RigidBody>
+</template>
+```
+
+## Types
+
+We can specify what kind of `RigidBody` type. `Dynamic` is the default.
```
Comment (alvarosabu) [inline docs/components/rigid-body.md:36]:
```suggestion
We can specify which kind of `RigidBody` type we want. By default is set to `Dynamic`(see below).
```

File: docs/components/rigid-body.md [hunk 7]
```diff
@@ -1,204 +1,230 @@
...
+      <TresPlaneGeometry :args="[20, 20, 20]" :rotate-x="-Math.PI / 2" />
+      <TresMeshBasicMaterial color="#f4f4f4" />
+    </TresMesh>
+  </RigidBody>
+</template>
+```
+
+### Available types
+
+| Prop                     | Description                                                                                                  |
```
Comment (alvarosabu) [inline docs/components/rigid-body.md:53]: Prop?

File: docs/components/rigid-body.md (hunk 1 above)
Comment (alvarosabu) [reply docs/components/rigid-body.md:227]: Agree

**Review by alvarosabu — CHANGES_REQUESTED** [review]

**Comment by alvarosabu** [comment]
This PR wasn't ready for merging. I reverted it, please re-open and apply the changes required.

File: docs/guide/how-does-it-work.md [hunk 8]
```diff
@@ -1,6 +1,175 @@
...
+  <IntroDemo />
+</DocsDemo>
+
+These physics are working, but the sphere is not behavior like an sphere, is not rolling, why is that?
+
+You can inspect your physic scene using the `debug` prop in the `<Physics>` component you should see something like this:
+
+![](/img/cuboid.jpg)
+
+You'll quickly realize that our 3D sphere is round by a cube not a sphere, let's introduce the Colliders.
```
[the remark this answers is not in the document]
Comment (Neosoulink) [reply docs/guide/how-does-it-work.md:152]: Yeah, it's something I planned at the very beginning, 

You can see the code in charge of creating the collider-desc:

```ts
  if (
    shape === 'ball'
    || (shape === undefined
      && object instanceof Mesh
      && (object.geometry instanceof SphereGeometry
        || object.geometry instanceof IcosahedronGeometry))
  ) {
    colliderDesc = ColliderDesc.ball(radius ?? 1)
  }
```

This means it's supported for custom colliders, but not yet for automatic colliders

File: docs/guide/how-does-it-work.md [hunk 9]
```diff
@@ -1,6 +1,175 @@
...
+
+Colliders are physic object with shape, properties and methods, they are child of the RigidBody component, a RigidBody can have one or many colliders. But we haven't create any, why that collider appears there?
+
+Well `@tresjs/rapier` will create an automatic collider for you, for each Mesh that you add! This normally is really good DX, but in cases like this need to tell our collider to have a sphere or `ball` shape.
```
[the remark this answers is not in the document]
Comment (Neosoulink) [reply docs/guide/how-does-it-work.md:156]: Yeah, it's something I planned at the very beginning, 

You can see the code in charge of creating the collider-desc:

```ts
  if (
    shape === 'ball'
    || (shape === undefined
      && object instanceof Mesh
      && (object.geometry instanceof SphereGeometry
        || object.geometry instanceof IcosahedronGeometry))
  ) {
    colliderDesc = ColliderDesc.ball(radius ?? 1)
  }
```

This means it's supported for custom colliders, but not yet for automatic colliders

**Pull request merged**